### PR TITLE
Add 2021Q2 entry for doceng

### PR DESCRIPTION
**Pull request by dbaio**
You can close/ignore #375 in favor of this one.

### CODE REVIEW

File: 2021q2/doceng.adoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
+=== Documentation Engineering Team
+
+Link: link:https://www.freebsd.org/docproj/[FreeBSD Documentation Project] +
+Link: link:https://docs.freebsd.org/en/books/fdp-primer/[FreeBSD Documentation Project Primer for New Contributors] +
+Link: link:https://www.freebsd.org/administration/#t-doceng[Documentation Engineering Team]
+
+Contact: FreeBSD Doceng Team <doceng@FreeBSD.org> +
+
+The doceng@ team is a body to handle some of the meta-project issues associated with the FreeBSD Documentation Project, more infomation on link:https://www.freebsd.org/internal/doceng/[FreeBSD Doceng Team Charter].
```
Comment (pauamma) [inline 2021q2/doceng.adoc:9]: s/body to handle/group handling/
s/infomation on/information in the/
Also, I'd change the "," before "more" to ";", but that may be a matter of taste.

File: 2021q2/doceng.adoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+During the last quarter, we welcomed back Ceri Davies (ceri@) as a doc committer. Sergio Carlavilla (carlavilla@) and Danilo G. Baio (dbaio@) joined the doceng@ team.
+
+A new article was included in the FreeBSD Documentation by Ka Ho Ng (khng@), link:https://docs.freebsd.org/en/articles/freebsd-src-lsp/[Use Language Servers for Development in the FreeBSD Src Tree].
+
+Many works were done regarding the transition to Hugo/Asciidoctor, but there are still pending issues in the FreeBSD Documentation Project after the migration.
```
Comment (pauamma) [inline 2021q2/doceng.adoc:15]: s/Many works were/Much work was/

File: 2021q2/doceng.adoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+The link:https://wiki.freebsd.org/Doc/IdeaList[TODO items] wiki list was refreshed, and everyone is welcome to contribute to it.
+
+==== FreeBSD Translations on Weblate
+
+Link: link:https://wiki.freebsd.org/Doc/Translation/Weblate[Translate FreeBSD on Weblate] +
+Link: link:https://translate-dev.freebsd.org/[FreeBSD Weblate Instance]
+
+The Weblate project link:https://translate-dev.freebsd.org/projects/documentation/[Documentation (Books and Articles)] is open to the public.
+
+The Automatic Translation feature was executed in all languages, re-using translations from the former project (Docbook).
```
Comment (pauamma) [inline 2021q2/doceng.adoc:26]: Not sure what you mean by "executed" here. Added? Implemented? Something else?

File: 2021q2/doceng.adoc (hunk 1 above)
Comment (dbaio) [reply 2021q2/doceng.adoc:9]: This text is part of  the [FreeBSD Doceng Team Charter](https://www.freebsd.org/internal/doceng/).

File: 2021q2/doceng.adoc (hunk 3 above)
Comment (dbaio) [reply 2021q2/doceng.adoc:26]: This is the feature: [Weblate Automatic translation](https://docs.weblate.org/en/latest/user/translating.html#automatic-translation)

Comment (pauamma) [reply 2021q2/doceng.adoc:26]: Thanks for the link, but I would still use something clearer in context than "executed". Perhaps one of the other possibe words I listed or perhaps something else, depending on whar you mean.

File: 2021q2/doceng.adoc (hunk 2 above)
Comment (dbaio) [reply 2021q2/doceng.adoc]: Thanks

File: 2021q2/doceng.adoc (hunk 3 above)
Comment (dbaio) [reply 2021q2/doceng.adoc:26]: This context is `execute/run` a script or command.

File: 2021q2/doceng.adoc [hunk 4]
```diff
@@ -12,7 +12,7 @@ During the last quarter, we welcomed back Ceri Davies (ceri@) as a doc committer
 
 A new article was included in the FreeBSD Documentation by Ka Ho Ng (khng@), link:https://docs.freebsd.org/en/articles/freebsd-src-lsp/[Use Language Servers for Development in the FreeBSD Src Tree].
 
-Many works were done regarding the transition to Hugo/Asciidoctor, but there are still pending issues in the FreeBSD Documentation Project after the migration.
+Much work was done regarding the transition to Hugo/Asciidoctor, but there are still pending issues in the FreeBSD Documentation Project after the migration.
 We thank everyone who is helping to find and fixing them.
```
Comment (pauamma) [inline 2021q2/doceng.adoc:16]: Missed that one on my first pass: "everyone helping find and fix them"

**Pull request closed without merging**